src/components/loader-bar/loader-bar.component.tsx

```diff
@@ -1,20 +1,44 @@
 import React from "react";
 import { MarginProps } from "styled-system";
-import tagComponent from "../../__internal__/utils/helpers/tags/tags";
+import { filterStyledSystemMarginProps } from "../../style/utils";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import useLocale from "../../hooks/__internal__/useLocale";
+import tagComponent, {
+  TagProps,
+} from "../../__internal__/utils/helpers/tags/tags";
+
 import StyledLoaderBar, {
   InnerBar,
   StyledLoader,
   StyledLoaderBarProps,
 } from "./loader-bar.style";
 
-export interface LoaderBarProps extends StyledLoaderBarProps, MarginProps {}
+export interface LoaderBarProps
+  extends StyledLoaderBarProps,
+    MarginProps,
+    TagProps {}
 
 export const LoaderBar = ({ size = "medium", ...rest }: LoaderBarProps) => {
+  const l = useLocale();
+
+  const reduceMotion = !useMediaQuery(
+    "screen and (prefers-reduced-motion: no-preference)"
+  );
+
   return (
-    <StyledLoader {...rest} {...tagComponent("loader-bar", rest)}>
-      <StyledLoaderBar size={size}>
-        <InnerBar size={size} />
-      </StyledLoaderBar>
+    <StyledLoader
+      aria-label={l.loader.loading()}
```

Comment:
sorry I didn't pick this up before - but it seems curious to me that we allow the consumer to override the `aria-label` on `Loader` but not on `LoaderBar`. This seems inconsistent. (I'm not actually sure we need to allow this to be overridden tbh, but there's no harm in it.)

Reply:
The `aria-label` prop for `Loader` already existed which is why I included it there but not with `LoaderBar` - to avoid adding a prop deprecation or introduce a prop which we don't really need if we use the locale variable

Reply:
Ah OK, fair enough - no need to add it now then